content/using/id/guide-to-breaches.md

```diff
@@ -6,27 +6,32 @@ template = "doc.html"
 
 An important concept on the [Ames](@/docs/arvo/ames/ames.md) network is that of continuity. Continuity refers to how ships remember the order of their own network messages and the network messages of others -- these messages are numbered, starting from zero. A _breach_ is when ships on the network agree to forget about this sequence and treat one or more ships like they are brand new.
 
-There are two kinds of breaches: **personal breaches** and **network breaches**.
+There are two kinds of breaches: (personal) **breaches** and **network breaches**.
 
-## Personal Breaches
+## Breaches
 
-Ships on the Ames network sometimes need to reset their continuity. A personal breach is when an individual ship announces to the network: "I forgot who I am, let's start over from scratch." That is, it clears its own event log and sends an announcement to the network, asking all ships that have communicated with it to reset its networking information in their state. This makes it as though the ship was just started for the first time again, since everyone on the network has forgotten about it.
+Ships on the Ames network sometimes need to reset their continuity. A breach is
+when an individual ship announces to the network: "I forgot who I am, let's
+start over from scratch." That is, it clears its own event log and sends an
+announcement to the network, asking all ships that have communicated with it to
+reset its networking information in their state. This makes it as though the
+ship was just started for the first time again, since everyone on the network
+has forgotten about it.
 
-Personal breaches often fix connectivity issues, but should only be used as a
-last resort. Before performing a personal breach, look at alternative fixes in
-the [Ship Troubleshooting](../ship-troubleshooting) guide. Also reach out for
-help in the Help channel in the Urbit Community group
-`~bitbet-bolbel/urbit-community`, or, failing that, in the
-`#ship-starting-support` channel in our [Discord
+Breaches often fix connectivity issues, but should only be used as a last
+resort. Before performing a breach, look at alternative fixes in the [Ship
+Troubleshooting](../ship-troubleshooting) guide. Also reach out for help in the
+Help channel in the Urbit Community group `~bitbet-bolbel/urbit-community`, or,
+failing that, in the `#ship-starting-support` channel in our [Discord
 server](https://discord.gg/n9xhMdz) to see if there is another option.
 
-There are two types of personal breaches - one where your Ethereum ownership address
+There are two types of breaches - one where your Ethereum ownership address
```

Comment:
Looking at this again, "two types of breaches" feels a little bit misleading. It's more like, "you can breach when you own a thing/change its keys, and you can also breach during a transfer of the thing". I don't think this changes much about the instructions below, they're still two separate _flows_, so maybe this is a bit nitpicky.